Lib/urllib/parse.py

```diff
@@ -434,6 +433,8 @@ def _checknetloc(netloc):
             raise ValueError("netloc '" + netloc + "' contains invalid " +
                              "characters under NFKC normalization")
 
+# Cache size chosen by random die roll; prior to functools it was 20 in 1997.
+@functools.lru_cache(maxsize=99)
```

Comment:
If it were me, I wouldn't specify a maxsize and just accept the default.   Also, there is *slight* advantage to making the maxsize a power of two.